aws/src/main/java/sunstone/aws/impl/AwsConfig.java

```diff
@@ -6,7 +6,7 @@
  */
 public class AwsConfig {
 
-    public static final String ACCESS_KEY_ID = "aws.accessKeyID";
-    public static final String SECRET_ACCESS_KEY = "aws.secretAccessKey";
-    public static final String REGION = "aws.region";
+    public static final String ACCESS_KEY_ID = "sunstone.aws.accessKeyID";
+    public static final String SECRET_ACCESS_KEY = "sunstone.aws.secretAccessKey";
+    public static final String REGION = "sunstone.aws.region";
```

Comment:
I assume it is because of `sunstone.properties` -> `microprofile.properties`. Because now property is global, so we need to specify that it is sunstone specific.

Reply:
Yes, the old implementation added `sunstone.` when it resolved a property. See https://github.com/wildfly-extras/sunstone/pull/153/files#diff-9486dfb33583c39fea0a853acc6e5f0c02b9191a02c2681126544aeae1e81db8L108